zkevm-circuits/src/evm_circuit/util/math_gadget/constant_division.rs

```diff
@@ -133,7 +122,7 @@ mod tests {
 
             cb.require_equal(
                 "correct reminder",
```

Comment:
Just spike a typo "remainder" :)

Reply:
Nice catch, fixed in [26b3527](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1525/commits/26b3527c028c204a758ab14222ff02e04b6b91d5)